pyrene/src/components/Table/TableCell/TableCell.tsx

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable react/prop-types */
+import React, { CSSProperties, ReactNode, FunctionComponent } from 'react';
+import clsx from 'clsx';
+
+import styles from './tableCell.css';
+
+export interface TableCellProps {
+  children: ReactNode,
+  multiSelect: boolean,
```

Comment:
I think `multiSelect` was not required before. Is this a mistake or a change made on purpose?